skrifa/src/outline/error.rs

```diff
@@ -84,3 +85,5 @@ impl fmt::Display for DrawError {
         }
     }
 }
+
+impl Error for DrawError {}
```

Comment:
the only possible concern here is that I think `Error` is only available via `std`, so I would remove the top-level import of the trait and do,

```suggestion
#[cfg(feature = "std")]
impl std::error::Error for DrawError {}
```

@dfrg does this sound right to you?

Reply:
I believe `Error` in `core` is relatively recent so we may want to stick with the optional `std` type until we confirm that our primary users are compiling with a suitable Rust version.

Reply:
MSRV is a bit squirrely, it's the floor of our major consumers and desired consumers so I don't actually know the authoritative answer.  That would lead me to be +1 on this suggestion:

```rust
#[cfg(feature = "std")]
impl std::error::Error for DrawError {}
```